frontend/src/APIClients/lessonAPIClient.ts

```diff
@@ -0,0 +1,89 @@
+import baseAPIClient from "./BaseAPIClient";
+import AUTHENTICATED_USER_KEY from "../constants/AuthConstants";
+import { getLocalStorageObjProperty } from "../utils/LocalStorageUtils";
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+export type LessonRequest = {
+  course: string;
+  title: string;
+  description: string;
+  image: string;
+  content: [Record<string, any>];
+};
+
+export type LessonResponse = {
+  id: string | number;
```

Comment:
won't it always be a string?

Reply:
I noticed that the [EntityResponse](https://github.com/uwblueprint/rowan-house/blob/main/frontend/src/APIClients/EntityAPIClient.ts#L21) type implemented it as string or number. 

But yeah, I would assume it'll always be a string so I'll remove the or number